ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/ospf-cleartext-passwords.yml

```diff
@@ -0,0 +1,275 @@
+---
+# Tests using cleartext OSPF passwords - verify they are automatically encrypted
+
+type: l3leaf
+
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 192.168.66.0/24
+    vtep_loopback_ipv4_pool: 192.168.142.0/24
+    virtual_router_mac_address: 00:11:22:33:44:55
+    bgp_as: 666
+  nodes:
+    - name: ospf-cleartext-passwords
+      id: 42
+      mgmt_ip: 192.168.42.42/24
+
+tenants:
+  - name: Tenant_OSPF
+    mac_vrf_vni_base: 10000
+    mac_vrf_id_base: 20000
+    redistribute_mlag_ibgp_peering_vrfs: true
+    vrfs:
+      - name: Tenant_OSPF_VRF_LEVEL_SIMPLE_ENCRYPTION
+        vrf_vni: 100
+        vrf_id: 42
+        ospf:
+          enabled: true
+          router_id: 10.10.10.10
+          # The VRF is configured with OSPF auth method simple
+          authentication: simple
+          cleartext_simple_auth_key: password
```

Comment:
`cleartext_simple_auth_key` is quite confusing key name. However if we don't want to rename existing `simple_auth_key` - I don't see a better choice. Maybe we can consider some naming clean-up in 6.0?

Reply:
not sure that will be a priority - changing model for the sake of changing the model is a burden on users. But feel free to log an action item